src/schemata/workflow-event-schema.yml

```diff
@@ -0,0 +1,15 @@
+---
+$schema: http://json-schema.org/draft-07/schema#
+title: Workflow Event
+description: A modification event done during a workflow execution.
+required:
+  - type
+  - payload
+oneOf:
+  - $ref: '#/definitions/workflow-event/addStep'
```

Comment:
So it you are literally using this file instead of the compiled JSON schema, then the snake/kebab casing may still affect you somehow maybe. Just pointing this out to make double-sure (is that a word?)

Reply:
Only the CircleCI pipeline will tell me once I run all the workflow tests 😸 
Let's see how it goes